packages/react/src/LogoSuite/LogoSuite.figma.tsx

```diff
@@ -2,612 +2,141 @@ import React from 'react'
 import {LogoSuite} from './LogoSuite'
 import figma from '@figma/code-connect'
 
-figma.connect(LogoSuite, 'https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=8553-10133&m=dev', {
-  props: {
-    description: figma.textContent('Description'),
-    heading: figma.textContent('Text'),
-    align: figma.enum('align', {
-      start: 'start',
-      center: 'center',
-    }),
+figma.connect(
+  LogoSuite,
+  'https://www.figma.com/design/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=14825-1968&t=R95Qkrf0BUjsetcA-11',
+  {
+    props: {
+      headingText: figma.boolean('heading?', {
+        true: figma.textContent('Heading'),
+        false: 'Heading',
+      }),
+      headingOpeningTag: figma.boolean('heading?', {
+        // eslint-disable-next-line github/unescaped-html-literal
```

Comment:
We should just disable `github/unescaped-html-literal` rule on all `*.figma.tsx` files. wdyt?